cli/cmd/root.go

```diff
@@ -284,8 +338,8 @@ func (options *proxyConfigOptions) flagSet(e pflag.ErrorHandling) *pflag.FlagSet
 	flags.StringVar(&options.imagePullPolicy, "image-pull-policy", options.imagePullPolicy, "Docker image pull policy")
 	flags.UintVar(&options.proxyInboundPort, "inbound-port", options.proxyInboundPort, "Proxy port to use for inbound traffic")
 	flags.UintVar(&options.proxyOutboundPort, "outbound-port", options.proxyOutboundPort, "Proxy port to use for outbound traffic")
-	flags.UintSliceVar(&options.ignoreInboundPorts, "skip-inbound-ports", options.ignoreInboundPorts, "Ports that should skip the proxy and send directly to the application")
-	flags.UintSliceVar(&options.ignoreOutboundPorts, "skip-outbound-ports", options.ignoreOutboundPorts, "Outbound ports that should skip the proxy")
+	flags.StringSliceVar(&options.rawIgnoreInboundPorts, "skip-inbound-ports", options.rawIgnoreInboundPorts, "Ports and/or port ranges that should skip the proxy and send directly to the application")
+	flags.StringSliceVar(&options.rawIgnoreOutboundPorts, "skip-outbound-ports", options.rawIgnoreOutboundPorts, "Outbound ports and/or port ranges that should skip the proxy")
```

Comment:
Might be worth to specify that these port ranges are inclusive of lower and upper bound. Off by one errors are the root of the rest of all evil :)

Reply:
Was thinking of using `Ports and/or port ranges (inclusive) ...` or `Ports and/or inclusive port ranges ...`.  Any preference?  My initial lean was toward the former.